assets/src/edit-story/components/library/test/mediaPane.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { arrange, mediaResponseWithPdfAndImage } from './_utils';
+
+describe('mediaPane', () => {
+  it('should only display supported mimeTypes and not crash on unsupported', async () => {
+    const { container } = await arrange({
+      mediaResponse: mediaResponseWithPdfAndImage,
+    });
+    expect(container.querySelector(`img[src*=testImage]`)).not.toBeNull();
```

Comment:
Could you change this to `expect(getByAltText('test-alt')).toBeDefined()`? That's a better test pattern.

Reply:
Right, alt text fix is not yet merged, so I added a small fix here.